From 21163035a690b73fac7867ebaca1d3c92fb332b0 Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Sun, 21 May 2023 18:03:42 +0300
Subject: [PATCH] [SW-145556] gaudi2: erom: bin: fix endianness

the erom need to written as le and not be like the others images.

Change-Id: Id3fd424b8dd419adfa51645ba020d7aadf98b6b5
Signed-off-by: Arie Brown <abrown@habana.ai>
---

diff --git a/board/habanalabs/common/post-image-spi-arc_global.sh b/board/habanalabs/common/post-image-spi-arc_global.sh
index 996a0a4..a8c6fba 100755
--- a/board/habanalabs/common/post-image-spi-arc_global.sh
+++ b/board/habanalabs/common/post-image-spi-arc_global.sh
@@ -105,6 +105,7 @@
 source ${DEVICE_BASE}/post-image-spi.sh
 
 ENDIANNESS="${ENDIANNESS:-be}"
+ENDIANNESS_EROM="${ENDIANNESS_EROM:-le}"
 
 function hl_get_ver {
 	FILE=$1
@@ -350,7 +351,7 @@
 
 	create_spi_image ${DEV_PREFIX}-spi-flash.bin.nocrc ${DEV_PREFIX}-security-arc.bin.${ENDIANNESS} \
 			 ${DEV_PREFIX}-images-pointers.bin.${ENDIANNESS} ${DEV_PREFIX}-preboot.bin.${ENDIANNESS} \
-			 ${DEV_PREFIX}-ppboot-pcie_phy_fw.bin.${ENDIANNESS} ${DEV_PREFIX}-erom.bin.${ENDIANNESS} \
+			 ${DEV_PREFIX}-ppboot-pcie_phy_fw.bin.${ENDIANNESS} ${DEV_PREFIX}-erom.bin.${ENDIANNESS_EROM} \
 			 ${DEV_PREFIX}-fake-fuse.bin.${ENDIANNESS} ${DEV_PREFIX}-erom_tpm.bin.${ENDIANNESS} \
 			 ${DEV_PREFIX}-${PIF_ICCM_BINFILE}.${ENDIANNESS} ${DEV_PREFIX}-${PIF_DCCM_BINFILE}.${ENDIANNESS}
 
